docs/src/main/sphinx/connector/table-redirection.fragment

```diff
@@ -0,0 +1,38 @@
+In the context of lakehouse connectors it is quite common to use the same
```

Comment:
"lakehouse connectors" is not a concept that we have defined in the docs anywhere.. so you need to enumerate them or whatever

Reply:
avoid "it is quite common" and other fillers